src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
```

Comment:
- [ ] Kindly make sure to add a reducer here that checks the status and always returns "Under Construction", as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md#:~:text=Includes%20a%20reducer%20that%20checks%20the%20status%20and%20always%20returns%20%22Under%20construction%22%20(the%20initial%20state%20should%20check%20to%20that%20string)) requirement.

- [ ] Please make sure to add an MIT license in the readme.